app/src/main/java/ru/vssemikoz/newsfeed/dialogs/PickCategoryDialog.java

```diff
@@ -0,0 +1,44 @@
+package ru.vssemikoz.newsfeed.dialogs;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+
+import ru.vssemikoz.newsfeed.models.Category;
+import ru.vssemikoz.newsfeed.utils.Enum;
+
+public class PickCategoryDialog extends DialogFragment {
+    public interface NoticeDialogListener {
```

Comment:
Why have you called it `NoticeDialogListener`? What kind of notice does it contain? Let's call it `OnCategorySelectedListener` and rename corresponding method to `onCategorySelected`.